src/context/context.ts

```diff
@@ -383,6 +387,20 @@ export class WorkflowContext<TInitialPayload = unknown> {
     }
   }
 
+  /**
+   * Notify waiting workflow runs
```

Comment:
I think this explanation should tell a bit more than the function signature itself. Since this is not trivial, can we give more explanation?